frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -74,13 +83,25 @@ const applicationsUserAdministers = computed(() => {
 
 const tabViewRef = ref();
 
-onUnmounted(() => {
+const resetNewTag = () => {
+    router.push({ query: {} });
+    newAppAdminId.value = '';
+    newUserAccessIds.value = '';
+    newDelegatedAdminIds.value = '';
```

Comment:
here I'm not so sure, we need to clean the query parameter for the router, but somehow I found we need to manually reset these local parameter values.

Reply:
This is interesting, I am curious, is a `push` actually fetch again?

Reply:
push for just query will not trigger the fetch again by trying locally